src/Workspaces/Core/Portable/Shared/TestHooks/IAsynchronousOperationWaiter.cs

```diff
@@ -10,7 +10,12 @@ internal interface IAsynchronousOperationWaiter
         bool TrackActiveTokens { get; set; }
         ImmutableArray<AsynchronousOperationListener.DiagnosticAsyncToken> ActiveDiagnosticTokens { get; }
 
-        Task CreateWaitTask();
```

Comment:
We've confirmed this isn't being used in any test harnesses anywhere else that is driving Roslyn? I think some of that is hidden behind the waiting service but we should confirm that.

Reply:
I have refactored this in the past without problems, so I'm assuming that hasn't changed.